content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -37,6 +37,11 @@ Configuring your installation with {{% config-file-istio-dex %}} has a few optio
   1.3.1 with SDS enabled, which requires you to use Kubernetes 1.13 or later. You may need
   to add extra configurations to your Kubernetes as mentioned in
   [Istio's blog](https://istio.io/blog/2019/trustworthy-jwt-sds/).
+  The Istio community runs their test-infrastructure with the following API Server flags:
+  ```
+  "service-account-issuer": "kubernetes.default.svc"
+  "service-account-signing-key-file": "/etc/kubernetes/pki/sa.key"
```

Comment:
@swiftdiaries I suggest that we do not suggest the values mentioned here as these are required to be configured based on a user's setting. The values mentioned here may not be applicable to all installations of Kubernetes.

If we chose to use these example values, let us highlight it with the wording:

"For example, the Istio community runs their [test-infrastructure] (link to docs for test-infrastructure with flags)...."